build/packager.js

```diff
@@ -257,15 +257,26 @@ Packager.prototype.package = function (next) {
 		}.bind(this),
 		// Now include all the pre-built node-ios-device bindings/binaries
 		function (cb) {
-			if (this.targetOS == 'osx') {
-				exec('node bin/download-all.js', {cwd: path.join(this.zipSDKDir, 'node_modules', 'node-ios-device')}, function (err, stdout, stderr) {
+			if (this.targetOS === 'osx') {
+				var dir = path.join(this.zipSDKDir, 'node_modules', 'node-ios-device');
+
+				if (!fs.existsSync(dir)) {
+				    dir = path.join(this.zipSDKDir, 'node_modules', 'ioslib', 'node_modules', 'node-ios-device');
+				}
+
+				if (!fs.existsSync(dir)) {
+				    return cb(new Error('Unable to find node-ios-device module'));
+				}
+
+				exec('node bin/download-all.js', {cwd: dir},  function (err, stdout, stderr) {
```

Comment:
I would just inherit stdio, then you don't need to manually print the stdout/stderr. You can just pass in `cb` like this:

```js
exec('node bin/download-all.js', { cwd: dir, stdio: 'inherit' }, cb);
```

Also note that we put spaces inside braces/brackets around the props/elements.